app/models/vm_or_template.rb

```diff
@@ -879,46 +879,22 @@ def my_zone_obj
   #
 
   # TODO: Come back to this
-  def proxies4job(job = nil)
+  def proxies4job(_job = nil)
     _log.debug "Enter"
-    proxies = []
-    msg = 'Perform SmartState Analysis on this VM'
-    embedded_msg = nil
-
-    # If we do not get passed an model object assume it is a job guid
-    if job && !job.kind_of?(ActiveRecord::Base)
-      jobid = job
-      job = Job.find_by(:guid => jobid)
-    end
```

Comment:
Wow, we do all that work with the job just for logging... Nice find.  We don't need it.